modules/bazel_env.bzl/0.1.2/presubmit.yml

```diff
@@ -0,0 +1,14 @@
+bcr_test_module:
+  module_path: "examples"
+  matrix:
+    platform: ["debian10", "macos", "ubuntu2004"]
+    bazel: ["7.x"]
+  tasks:
+    run_tests:
+      name: "Run test module"
```

Comment:
Adding 
```
    environment:
      JAVA_TOOL_OPTIONS: ""
```
could probably help

Reply:
Even an empty value is picked up. Is there syntax to fully remove an environment variable? If not, I could look into adding it.

Reply:
OK, then maybe set `--test_env=JAVA_TOOL_OPTIONS=`, and hopefully it'll override the one set by CI.

Reply:
Oh, I see, it's not the override didn't work, it's an empty value was actually picked up.. OK, then I have no idea how this works underlying.

Reply:
I sent https://github.com/bazelbuild/continuous-integration/pull/2029, do you think that could work here?